frontend/src/metabase/styled-components/containers/GlobalStyles/GlobalStyles.tsx

```diff
@@ -41,6 +41,7 @@ export const GlobalStyles = (): JSX.Element => {
     ${aceEditorStyles}
     ${saveDomImageStyles}
     body {
+      font-size: 0.875em;
```

Comment:
If we keep `font-size` inside `base.styled.ts` then we don't need to add this, but I can understand the intention.